commit 5983deb4da2fe1a012b4cc13b777a3514a3e3b5f (refs/changes/97/4497/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-07-09T19:19:00-04:00 (1 year, 3 months ago)
    
    joyent/node-assert-plus#32 package.json repository does not point to Joyent

diff --git a/package.json b/package.json
index 40d6a5c..9cb0e2d 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/mcavage/node-assert-plus.git"
+    "url": "https://github.com/joyent/node-assert-plus.git"
   },
   "engines": {
     "node": ">=0.8"
